lib/src/ioctl/qbuf.rs

```diff
@@ -164,6 +164,11 @@ impl<H: PlaneHandle> Default for QBuffer<H> {
 }
 
 impl<H: PlaneHandle> QBuffer<H> {
+    pub fn set_timestamp(mut self, sec: i64, usec: i64) -> Self {
```

Comment:
The `timestamp` member is public (because any value is valid anyway), so it should be modifiable as long as you have a mutable reference to the `QBuffer`, doesn't that do the trick?

Reply:
Yes, this method is not necessary to set the `timestamp` member of the struct. I thought of this as a convenience method to avoid having to explicitly import `TimeVal` and add the dependency with the `nix` crate only for that. Using this method, one can initialize the timestamp without having to be aware of the underlying type.

Maybe it should have another name. Or maybe you see no value in having it, which would be ok too. But it is not meant as a mandatory setter for a private member, but a complementary one, keeping the member public.